modules/migration-network-traffic-redirection-strategies.adoc

```diff
@@ -0,0 +1,50 @@
+
+[id="migration-network-traffic-redirection-strategies_{context}"]
+= Network traffic redirection strategies
+
+After a successful migration, you must redirect network traffic of your stateless applications from the source cluster to the target cluster.
+
+The methods for redirecting network traffic are based on the following assumptions:
+
+* The application pods are running on both the the source and target clusters.
+* Each application has a route that contains the source cluster hostname.
+* The route with the source cluster hostname contains a CA certificate.
+* If you require secure connections through the source cluster's DNS domain, the target router default certificate includes the source cluster wildcard `Subject Alternative Name`.
+
+Consider the following methods and select the one that meets your objectives.
+
+* Redirecting all network traffic for all applications at the same time
++
+Change the wildcard DNS record of the source cluster to point to the target cluster router's virtual IP address (VIP).
++
+This method is suitable for simple applications or small migrations.
+
+* Redirecting network traffic for individual applications
++
+Create a DNS record for each application with the source cluster hostname pointing to the target cluster router's VIP. This DNS record takes precedence over the source cluster wildcard DNS record.
+
+* Redirecting network traffic gradually for individual applications
+
+. Create a proxy that can direct traffic to both the source cluster router's VIP and the target cluster router's VIP, for each application.
+
+. Create a DNS record for each application with the source cluster hostname pointing to the proxy.
+
+. Configure the proxy entry for the application to route a percentage of the traffic to the target cluster router's VIP and the rest of the traffic to the source cluster router's VIP.
+
+. Gradually increase the percentage of traffic that you route to the target cluster router's VIP until all the network traffic is redirected.
+
+* User-based redirection of traffic for individual applications
++
+Using this method, you can filter TCP/IP headers of user requests to redirect network traffic for predefined groups of users. This allows you to test the redirection process on specific populations of users before redirecting the entire network traffic.
+
+. Create a proxy that can direct traffic to both the source cluster router's VIP and the target cluster router's VIP, for each application.
+
+. Create a DNS record for each application with the source cluster hostname pointing to the proxy.
+
+. Configure the proxy entry for the application to route traffic matching a given header pattern, such as `test customers`, to the target cluster router's VIP and the rest of the traffic to the source cluster router's VIP.
+
+. Redirect traffic to the target cluster VIP in stages until all the traffic is on the target cluster router's VIP.
```

Comment:
"target cluster VIP" > "target cluster router's VIP" for consistence. It's a bit verbose but I'm not sure I can come up with an alternative.